src/mds/MDCache.cc

```diff
@@ -5524,8 +5524,15 @@ void MDCache::export_remaining_imported_caps()
     }
   }
 
+  for (map<inodeno_t, list<MDSInternalContextBase*> >::iterator p = cap_reconnect_waiters.begin();
+       p != cap_reconnect_waiters.end();
+       ++p)
+    mds->queue_waiters(p->second);
```

Comment:
Mmm, true, but those just get buried in the mds log and don't look like a client reconnect bug. Anything remaining here is a specific, special category of unknown ino, right?

Maybe we don't need anything special flagging it but such things make me nervous.